Modelica/Mechanics/MultiBody/Sensors/Internal/PartialAbsoluteBaseSensor.mo

```diff
@@ -5,17 +5,18 @@ model PartialAbsoluteBaseSensor
 
   Modelica.Mechanics.MultiBody.Interfaces.Frame_a frame_a
     "Coordinate system from which kinematic quantities are measured" annotation (Placement(
-        transformation(extent={{-116,-16},{-84,16}})));
+        transformation(extent={{-116,-16},{-84,16}})),
+      mustBeConnected="Connector frame_a should be connected");
 
   Modelica.Mechanics.MultiBody.Interfaces.Frame_resolve frame_resolve
     "Coordinate system in which output vector(s) is optionally resolved"
     annotation (Placement(transformation(extent={{-16,-16},{16,16}},
         rotation=-90,
-        origin={0,-100})));
+        origin={0,-100})),
+      mustBeConnected="Connector frame_resolve should be connected",
+      mayOnlyConnectOnce="Connector frame_resolve must be connected exactly once");
```

Comment:
Also
```suggestion
      mustBeConnected="Connector frame_resolve should be connected");
```
As discussed in https://github.com/modelica/ModelicaStandardLibrary/issues/4225#issuecomment-1798133364